dhis-2/dhis-support/dhis-support-artemis/src/main/java/org/hisp/dhis/artemis/audit/AuditableEntity.java

```diff
@@ -38,5 +38,19 @@
 @AllArgsConstructor
 public class AuditableEntity
 {
-    private Object entity;
+    /**
+     * Class of the AuditableEntity
+     * It will be used by {@link org.hisp.dhis.artemis.audit.legacy.AuditObjectFactory#collectAuditAttributes(AuditableEntity)} (Object)}
+     */
+    Class entityClass;
+
+    /**
+     * An object that is ready for serialized by Jackson.
+     * Means that this object should:
+     * 1. Only includes referenced properties that are owned by the current Audit Entity. Means that the property's schema has attribute "owner = true"
+     * 2. Do not include any lazy HibernateProxy or PersistentCollection that is not loaded.
+     * 3. All referenced properties that extend BaseIdentifiableObject should be mapped to only UID string
+     * This object could be a Map<String, Object>  with key is property name and value is the property value
+     */
+    Object serializableObject;
```

Comment:
I couple of questions:

1. Why Object? You say it can be Map<String, Object>, what is the reason for not doing it? Could it be Serializable (Since name is serializeableObject)?
2. Could it have a more descriptive name? To me, I think entity is a better name?

Reply:
resolved.